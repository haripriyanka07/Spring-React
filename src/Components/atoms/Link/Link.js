import { Link } from "react-router-dom";
import './Link.css';

const link = (props) => {
    return (
        <Link className="link" to={props.to}>{props.children}</Link>
    )
}
export default link;