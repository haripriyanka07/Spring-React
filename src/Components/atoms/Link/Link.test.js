import React from 'react';
import { BrowserRouter as Router} from "react-router-dom";
import renderer from 'react-test-renderer';
import Link from './Link';

it('test the input', () => {
    const renderedComponent = renderer.create(<Router><Link to='linkstate'>newLink</Link></Router>).toJSON();
    expect(renderedComponent).toMatchSnapshot();
})