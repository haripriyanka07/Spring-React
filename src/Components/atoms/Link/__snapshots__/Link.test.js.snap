// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the input 1`] = `
<a
  className="link"
  href="/linkstate"
  onClick={[Function]}
>
  newLink
</a>
`;
